MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -218,6 +218,8 @@ public override void EndScreenDeviceChange(string screenDeviceName, int clientWi
                 Sdl.Window.SetFullscreen(Handle, (_willBeFullScreen) ? fullscreenFlag : 0);
                 _hardwareSwitch = _game.graphicsDeviceManager.HardwareModeSwitch;
             }
+            // If going to exclusive full-screen mode, force the window to minimize on focus loss
+            Sdl.SetHint("SDL_VIDEO_MINIMIZE_ON_FOCUS_LOSS", _willBeFullScreen && _hardwareSwitch ? "1" : "0");
```

Comment:
Add a conditional to only apply that to Windows, on Linux I can just manually minimize the game and I get the resolution back to the original.